.eslintrc.js

```diff
@@ -3,8 +3,10 @@
 // or overridden for a particular subset of the project.  See
 // other .eslintrc.js files for those rules.
 module.exports = {
+  "parser": "babel-eslint",
```

Comment:
I don't understand what this does - do we now lint _while_ parsing as part of our build or something?  If so, should we remove our other linting steps?

Have you taken the [known issues with babel-eslint and Flow](https://github.com/babel/babel-eslint#known-issues) into account?